dotCMS/src/main/java/com/dotcms/ai/api/EmbeddingsAPIImpl.java

```diff
@@ -382,39 +307,99 @@ public void initEmbeddingsTable() {
      * @return Tuple(Count of Tokens Input, List of Embeddings Output)
      */
     @Override
-    public Tuple2<Integer, List<Float>> pullOrGenerateEmbeddings(@NotNull String content) {
+    public Tuple2<Integer, List<Float>> pullOrGenerateEmbeddings(@NotNull final String content) {
```

Comment:
missing db annotation

Reply:
We probably need them on most all of these methods